### PR TITLE
adding debriefing blog post

### PR DESCRIPTION
**Pull request by k8hertweck**

### CODE REVIEW

File: blog/2015/08/instructor-debriefing-2015-08-07.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<!-- end excerpt -->
+<h2>Interesting ideas to consider</h2>
+<p>The experienced instructors in our debriefing session compared recent workshops and correpsonding materials with their previous teaching experiences. Discussions with these folks are especially important given their comparatively large sample size of how students respond to certain approaches. In particular, we talked about the benefits of slowly introducing concepts in Python and R compared to jumping right into advanced topics and explaining mechanisms as the lessons progressed. For example, the <a href="https://github.com/swcarpentry/r-novice-gapminder">R gapminder lesson</a> can start with impressive, powerful packages like dplyr and tidyR to motivate students. However, this can be confusing to students new to programming, and will require backtracking later in the lesson to explain more basic concepts. This problem is can sometimes be alleviated by appropriately contextualizing the point of the lesson (i.e., automation of work, rather than learning how every algorithm works). Conversations about these issues are ongoing on GitHub and in various listservs.</p>
```
Comment (rgaiacs) [inline blog/2015/08/instructor-debriefing-2015-08-07.html]: Can we have a link to https://github.com/swcarpentry/python-novice-inflammation/issues/127?

File: blog/2015/08/instructor-debriefing-2015-08-07.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<p>Instructors for one workshop reported that a few students attended both days of the the workshop without laptops. The students discussed material together and took notes, and seemed surprisingly satisfied with their experience. Another option is to pair students without a machine with another student who can share their screen.</p>
```
Comment (rgaiacs) [inline blog/2015/08/instructor-debriefing-2015-08-07.html]: Can you mention that this workshop took place at SciPy and this was the reason that a few students attended without laptops?

File: blog/2015/08/instructor-debriefing-2015-08-07.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+<h2>What was difficult</h2>
+<p>Figuring out how to trim down copious lesson materials to fit the alloted time for a class continues to be problematic, especially for the Python and R materials (but see below). We recommend that experienced instructors partnered with new instructors for upcoming workshops share their thoughts about how to cut out sections of lessons.</p>
```
Comment (rgaiacs) [inline blog/2015/08/instructor-debriefing-2015-08-07.html]: Can you mention that new instructors could contact us at mentoring@lists.software-carpentry.org or workshop coordinators requesting introductions to a "senior" instructor for a conversation?

**Comment by rgaiacs** [comment]
@k8hertweck Great job. :sparkles: Thanks a lot.

**Comment by k8hertweck** [comment]
Thanks for the suggestions, @rgaiacs 
All changes have been added to the blog post.

**Pull request merged**